integration/test/app/basic/simple_test.go

```diff
@@ -3,21 +3,21 @@
 package basic
 
 import (
+	"context"
 	"fmt"
-	"golang.org/x/net/context"
 	"testing"
 
 	"github.com/giantswarm/e2e-harness/pkg/release"
 	"github.com/giantswarm/e2etemplates/pkg/chartvalues"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/app-operator/integration/key"
 )
 
 const (
-	namespace                 = "giantswarm"
-	customResourceReleaseName = "apiextensions-app-e2e-chart"
-	chartOperatorVersion      = "chart-operator.giantswarm.io/version"
-	testAppReleaseName        = "kubernetes-test-app-chart"
-	testAppCatalogReleaseName = "test-app-catalog"
+	namespace            = "giantswarm"
+	chartOperatorVersion = "chart-operator.giantswarm.io/version"
+	testAppCatalogName   = "test-app-catalog"
```

Comment:
I renamed this from `testAppCatalogReleaseName` because it isn't actually a release name. More of these constants can move to the key package but I didn't want to make the diff too noisy.